tests/install/test_builder.py

```diff
@@ -205,3 +242,40 @@ def _some_long_io_op(_):
     build._threaded_map(_some_long_io_op, [])
     end = time.time()
     assert end - start < sleep_duration
+
+
+def test_correct_pt_variant_os():
+    # Check that all Linux variants return Linux
+    for linux_variant in build.OperatingSystem.LINUX.value:
+        os_ = build.OperatingSystem.from_str(linux_variant)
+        assert isinstance(
+            build.choose_pt_variant(os_, "x86_64", "cpu", RAI_VERSIONS.torch),
+            build._PTArchiveLinux,
+        )
+    # Check that ARM64 and X86_64 Mac OSX return the Mac variant
+    all_archs = (build.Architecture.ARM64, build.Architecture.X64)
+    for arch in all_archs:
+        os_ = build.OperatingSystem.DARWIN
+        assert isinstance(
+            build.choose_pt_variant(os_, arch, "cpu", RAI_VERSIONS.torch),
+            build._PTArchiveMacOSX,
+        )
+
+
+def test_PTArchive_MacOSX_url():
+    os_ = build.OperatingSystem.DARWIN
+    arch = build.Architecture.X64
+    pt_version = RAI_VERSIONS.torch
+
+    pt_linux_cpu = build._PTArchiveLinux(build.Architecture.X64, "cpu", pt_version)
+    x64_prefix = "https://download.pytorch.org/libtorch/"
+    assert x64_prefix in pt_linux_cpu.url
+
+    pt_macosx_cpu = build._PTArchiveMacOSX(build.Architecture.ARM64, "cpu", pt_version)
+    arm64_prefix = "https://github.com/CrayLabs/ml_lib_builder/releases/download/"
+    assert arm64_prefix in pt_macosx_cpu.url
+
+
+def test_PTArchive_MacOSX_gpu_error():
```

Comment:
ditto looks like class name changed. Maybe change test name to
```suggestion
def test_PTArchiveMacOSX_gpu_error():
```
or
```suggestion
def test_pt_archive_macosx_gpu_error():
```